bpfman-rpc/Cargo.toml

```diff
@@ -0,0 +1,77 @@
+[package]
```

Comment:
I'm not convinced we need to add yet-another-crate here.
Given the only consumer of `bpfman-api` is going to be `bpfman-rpc`... why not just add `bpfman-api/src/bin/main.rs`?

Reply:
Sounds good to me!

Reply:
uhh cyclic dependency hell, trying to clean this up